uploads/test_lights.txt

```diff
@@ -0,0 +1 @@
+ERROR: json to txt conversion not implemented yet
```

Comment:
Same as before. Why is this file here? It should likely be removed or gitignored.

Reply:
removed the file